src/doc/src/reference/build-scripts.md

```diff
@@ -147,13 +147,16 @@ linker script.
 
 [link-arg]: ../../rustc/codegen-options/index.md#link-arg
 
-### `cargo::rustc-cdylib-link-arg=FLAG` {#rustc-cdylib-link-arg}
+### `cargo::rustc-link-arg-cdylib=FLAG` {#rustc-cdylib-link-arg}
```

Comment:
Do we want to maintain the old anchor link?
I am fine wit either way.

Reply:
Yea, I prefer to keep the old anchor, just to avoid breaking any potential links.